md-dao/src/main/java/com/wipro/ats/bdre/md/dao/StepDAO.java

```diff
@@ -42,11 +42,13 @@
     private static final Logger LOGGER = Logger.getLogger(StepDAO.class);
     @Autowired
     SessionFactory sessionFactory;
-
+    private static final String PROCESS="process";
+    private static final String EXECSTATUS="execStatus";
+    private static final String BATCHBYTARGETBATCHID="batchByTargetBatchId";
     public Long initStep(Integer subPid) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "initStep" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)